compiler/rustc_codegen_llvm/src/base.rs

```diff
@@ -150,6 +150,19 @@ pub fn set_link_section(llval: &Value, attrs: &CodegenFnAttrs) {
     }
 }
 
+pub fn set_variable_sanitizer_attrs(llval: &Value, attrs: &CodegenFnAttrs) {
+    if attrs.no_sanitize.contains(SanitizerSet::ADDRESS) {
+        unsafe {
+            llvm::LLVMRustSetNoSanitizeAddress(llval);
+        }
+    }
+    if attrs.no_sanitize.contains(SanitizerSet::HWADDRESS) {
+        unsafe {
+            llvm::LLVMRustSetNoSanitizeHWAddress(llval);
+        }
```

Comment:
Tiny style nit; putting the semi outside the `{ ... }` braces for single-line unsafe code makes it format nicer